manifests/bosh-manifest/spec/fixtures/bosh-secrets.yml

```diff
@@ -1,6 +1,5 @@
 ---
 secrets:
-  bosh_postgres_password: BOSH_POSTGRES_PASSWORD
```

Comment:
This is currently being generated by `generate-bosh-secrets.rb`. If it's genuinely not used, then it should be removed from there as well, otherwise we should keep this fixture in sync with the generator.